libr/util/overflow.c

```diff
@@ -0,0 +1,11 @@
+/* radare - LGPL - Copyright 2016 - condret */
+
+#include <r_util.h>
+#include <r_types.h>
+
+R_API ut64 r_chk_overflow_add_ut64 (ut64 a, ut64 b)
```

Comment:
hm, I know, that this is already done, but there was something not so nice in the other implemention